pypyr/steps/pype.py

```diff
@@ -33,6 +33,8 @@ def run_step(context):
                 - useParentContext. optional. bool. Defaults to True. Pass the
                   current (i.e parent) pipeline context to the invoked (child)
                   pipeline.
+                - loader: str. optional. Path to the pipeline loader module.
```

Comment:
Absolute name of pipeline loader module. If not specified will use pypyr.pipeloaders.fileloader.